crates/matrix-sdk-base/src/rooms/mod.rs

```diff
@@ -100,10 +98,8 @@ pub struct BaseRoomInfo {
     pub(crate) tombstone: Option<MinimalStateEvent<RoomTombstoneEventContent>>,
     /// The topic of this room.
     pub(crate) topic: Option<MinimalStateEvent<RoomTopicEventContent>>,
-    /// All Minimal state events that containing one or more running matrixRTC
-    /// memberships.
-    #[serde(skip_serializing_if = "BTreeMap::is_empty", default)]
-    pub(crate) rtc_member: BTreeMap<OwnedUserId, MinimalStateEvent<CallMemberEventContent>>,
+    /// Stores the information about the current call memberships in a room.
+    pub(crate) call_memberships: CallMembershipsHandler,
```

Comment:
here I dont see `call_` to be fitting. What do you think about a list of `RTCSessionHandler`s.
Or one `RTCSessionsHandler` that provides all sessions.

Reply:
I did not use any "RTC session" wording in this PR because it does not occur on a `ruma` or state-event level. I.e. this particular object that contains `call_memberships` forwards `SyncStateEvent::CallMember(event)` to the `call_memberships`, i.e. the terminology of the RTC session does not happen here, so it would be confusing.

That's why I'm more for changing the terminology **once** and for all places once we decide to make it consistent (you probably remember how we tackled it in the Widget API where we had "permissions" and "capabilities" but at the end, there was one global rename to keep terminology the same).

That's why I avoid the term "RTC session" up until we decide to perform this rename globally (in SDK, in `ruma`, but also in spec! our spec contains `m.call.member` events, not `m.session.member` 🙂 ).

Reply:
I think changing it in the spec is sadly unrealistic... I discussed it a couple of times.

But on the sdk level we always used session wording. (in ruma the concept of a session just never occurs -> no session wording)

But in the js sdk we do use session whenever its a session. and rtc whenever its rtc (never call).

So basically the only place where we use call is for things directly referring to the sepc. So if we actually rename the spec only those occasions would need to be changed.